README.md

```diff
@@ -1 +1,34 @@
-# portfolio
\ No newline at end of file
+# ![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio Skeleton
+
+> This is my first portfolio.
+> It was based on a FIGMA template and I used HTML,
+> CSS and CSS FLEXBOX to build the first part of the mobile version.
+
```

Comment:
In order to have a more professional repo, it is better if we don't exclude sections from README.md file.

- [ ] Kindly, add back the following sections:
`Getting Started`
`Prerequisites` - what the user needs on his machine to reproduce your project
`Setup` - help with guided steps a non-professional user to clone your repo on his machine
`Install` - explain how  a non-professional user can run your project on his device


**After this, definitely, your repo will look professional** :1st_place_medal: